api/src/controllers/map-info-controller.js

```diff
@@ -0,0 +1,220 @@
+// const mongoose = require('mongoose');
+// const ObjectID = require('mongodb').ObjectID;
```

Comment:
Remove since they aren't used

Reply:
Fixed in https://github.com/bcgov/NRPTI/pull/790/commits/e14e75a6eafb62e6935dd5b1f8639c3561a8886c